projects/packages/search/src/smart-inline-search/class-inline-search.php

```diff
@@ -0,0 +1,409 @@
+<?php
+/**
+ * Inline Search: search without popup using v1.3 Instant Search API
+ *
+ * @package automattic/jetpack-search
+ */
+
+namespace Automattic\Jetpack\Search;
+
+/**
+ * Inline Search class
+ */
+class Inline_Search extends Classic_Search {
```

Comment:
should we change the "smart-inline-search" directory for this file also?

Reply:
Good point, I missed that when renaming the class. I've renamed the directory to "inline-search" now.